app/graphql/schema.graphql

```diff
@@ -25,6 +25,7 @@ type DataCube {
   version: String
   contactName: String
   contactEmail: String
+  creator: DataCubeOrganization
```

Comment:
If it makes sense, no problem for me to rename the field.

Reply:
Not sure if it really makes sense but I guess someone decided it for a reason. Maybe a super-detail thing to consider is that IF there are multiple creators attached to a cube, there's no defined order to them, so if we decide to pick the first one, it may be a bit of a random choice